Content.Server/GameTicking/Rules/TraitorRuleSystem.cs

```diff
@@ -184,6 +184,9 @@ public bool MakeTraitor(IPlayerSession traitor)
 
         var startingBalance = _cfg.GetCVar(CCVars.TraitorStartingBalance);
 
+        if (mind.CurrentJob != null)
+            startingBalance -= mind.CurrentJob.Prototype.DifficultyAdjustment;
```

Comment:
Clamp this so that if a difficulty adjustment is larger than the actual total balance it just becomes zero.